src/api/entities/Asset/types.ts

```diff
@@ -57,5 +57,16 @@ export interface AgentWithGroup {
   group: KnownPermissionGroup | CustomPermissionGroup;
 }
 
-export * from './CorporateActions/types';
+export interface AssetMetadataSpec {
+  url?: string;
+  description?: string;
+  typeDef?: string;
+}
+export interface AssetGlobalMetadata {
+  id: BigNumber;
+  name: string;
+  specs?: AssetMetadataSpec;
```

Comment:
```suggestion
  specs: AssetMetadataSpec;
```
I think specs are required even if every field might be `None`.